Dockerfile.local

```diff
@@ -28,6 +28,10 @@ RUN curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/
 
 ENV PATH "$PATH:/usr/local/bin/aws-iam-authenticator"
 
+RUN wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py && \
```

Comment:
Not familiar with pip at all or whether this matters, but it looks like in Clouddriver we are specifying a certain version of `awscli`?
https://github.com/spinnaker/clouddriver/blob/master/Dockerfile.slim#L23

Reply:
Good point, @maggieneterval! Probably best to pin this to a specific version to have a more predictable behavior, in cases something breaks with future versions. I made the change.